src/app/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,72 @@
+import classNames from 'classnames';
+import React, { useCallback, useMemo } from 'react';
+import styles from './index.module.scss';
+
+export enum CheckboxAlignment {
+  LEFT = 'left',
+  RIGHT = 'right',
+}
+
+type CheckboxProps = {
+  label: string | React.ReactNode;
+  checked: boolean;
+  onChange: () => void;
+  disabled?: boolean;
+  inline?: boolean;
+  alignment?: CheckboxAlignment;
+  className?: string;
+};
+
+export const Checkbox: React.FC<CheckboxProps> = ({
+  label,
+  checked,
+  onChange,
+  disabled = false,
+  inline = false,
+  alignment = CheckboxAlignment.LEFT,
+  className,
+}) => {
+  const onClick = useCallback(
+    e => {
+      e.preventDefault();
+      !disabled && onChange();
```

Comment:
Please just use an `if`. Also please pass the new value to onChange.

Reply:
There is no need for passing values, `checked` is controlled by a parent component (but good catch with `if`).